influxql/parser.go

```diff
@@ -493,6 +503,63 @@ func (p *Parser) parseSelectStatement() (*SelectStatement, error) {
 	return stmt, nil
 }
 
+// targetRequirement specifies whether or not a target clause is required.
+type targetRequirement int
+
+const (
+	targetRequired targetRequirement = iota
+	targetNotRequired
+)
+
+// parseTarget parses a string and returns a Target.
+func (p *Parser) parseTarget(tr targetRequirement) (*Target, error) {
+	if tok, pos, lit := p.scanIgnoreWhitespace(); tok != INTO {
+		if tr == targetRequired {
+			return nil, newParseError(tokstr(tok, lit), []string{"INTO"}, pos)
+		}
+		p.unscan()
+		return nil, nil
+	}
+
+	// Parse identifier.  Could be policy or measurement name.
```

Comment:
Old habits die hard.  I learned to type, on a typewriter, from a teacher born before the Great Depression...back when it was common to put 2 spaces after a period.  :)